src/ui/button/props.js

```diff
@@ -106,32 +105,38 @@ export type ButtonProps = {|
     scopes : $ReadOnlyArray<string>,
     responseType : string,
     billingOptions : BillingOptions,
-    state? : string
-|};
-
+    state? : string,
+    cspNonce? : {|
+        nonce? : string
+    |},
+    returnurl? : string
+   |};
+   
 export type ButtonPropsInputs = {|
     clientID : string,
     fundingSource? : $Values<typeof FUNDING>,
     style? : ButtonStyleInputs | void,
     locale? : $PropertyType<ButtonProps, 'locale'> | void,
     env? : $PropertyType<ButtonProps, 'env'> | void,
-    meta? : $PropertyType<ButtonProps, 'meta'> | void,
+    // meta? : $PropertyType<ButtonProps, 'meta'> | void,
     stage? : $PropertyType<ButtonProps, 'stage'> | void,
     stageUrl? : $PropertyType<ButtonProps, 'stageUrl'> | void,
     platform? : $PropertyType<ButtonProps, 'platform'> | void,
     authButtonSessionID? : $PropertyType<ButtonProps, 'sessionID'> | void,
     sessionID? : $PropertyType<ButtonProps, 'sessionID'> | void,
-    nonce? : string,
+    nonce : string,
+    responseType : string,
     csp? : {|
         nonce? : string
     |},
-    content? : ButtonContent,
+    displayLabel? : boolean,
+    customLabel : string,
     onClick? : OnClick
-|};
+  |};
 
 export const DEFAULT_STYLE = {
-    COLOR:  BUTTON_COLOR.BLUE,
-    SHAPE:  BUTTON_SHAPE.RECT
+    COLOR:  BUTTON_COLOR.GOLD,
```

Comment:
Should this be blue since gold isn't supported?

```suggestion
    COLOR:  BUTTON_COLOR.BLUE,
```

Reply:
done